### PR TITLE
Ce/duplicate usernames

### PR DESCRIPTION
**Pull request by calellowitz**
@emord cc: @esoergel @nickpell @snopoke 
http://manage.dimagi.com/default.asp?233608#1194482
introduced in https://github.com/dimagi/commcare-hq/pull/12636/files

### CODE REVIEW

File: corehq/apps/users/tests/bulk_upload.py [hunk 1]
```diff
@@ -228,3 +228,20 @@ def test_check_duplicate_usernames(self):
...
+            {
+                u'username': u'goodbye',
+                u'user_id': u'other id',
+            },
+        ]
+
+        try:
+            check_duplicate_usernames(user_specs)
+        except UserUploadError:
+            self.fail('UserUploadError incorrectly raised')
```
Comment (dimagimon) [inline corehq/apps/users/tests/bulk_upload.py]: W292 no newline at end of file

File: corehq/apps/users/bulkupload.py [hunk 2]
```diff
@@ -79,8 +79,9 @@ def check_duplicate_usernames(user_specs):
             duplicated_usernames.add(username)
         usernames.add(username)
 
-    raise UserUploadError(_("The following usernames have duplicate entries in "
-        "your file: " + ', '.join(duplicated_usernames)))
```
Comment (esoergel) [inline corehq/apps/users/bulkupload.py]: What's going on here, exactly?

**Comment by esoergel** [comment]
Looks g2m on build

**Comment by emord** [comment]
killed build on the lint commit. Previous commit passed tests

**Pull request merged**